src/styles/pages/_letter.scss

```diff
@@ -38,20 +65,104 @@
   text-align: justify;
 }
 
+.letter-page {
+  width: 80%;
+  max-width: 800px;
+  height: 1000px;
+  padding: 80px;
+  border: solid black 1px;
+  margin: 40px 80px 80px 80px;
+
+  h1 {
+    font-size: 24px;
+  }
+
+  h3 {
+    font-size: 12px;
+    margin-top: 2em;
+    margin-bottom: 30px;
+  }
+
+  .addresses {
+    margin-top: 1.5em;
+    padding-bottom: 1.5em;
+  }
+
+  .sender-address, .sender-address * {
+    font-size: 12px;
+  }
+
+  .receiver-address, .receiver-address * {
+    font-size: 16px;
+  }
+
+  *, .letter.letter * {
+    font-size: 16px;
+  }
+
+  .letter p {
+    margin-top: 7px;
+    margin-bottom: 7px;
+  }
+
+  .letter-greeting {
+    margin-bottom: 25px;
+  }
+
+  .letter span {
+    margin-top: 3px;
+    margin-bottom: 3px;
+  }
+
+  .letter-logo {
+    position: absolute;
+    top: 0.5em;
+    right: 0.5em;
+    width: 2.93cm;
+  }
+
+  .letter-footer {
+    margin-top: 50px;
+    page-break-after: always;
+  }
+}
+
 @media print {
+  /* Remove url and date from top and bottom of printed page */
+  @page {
+     size: auto;
+     margin: 0mm;
+   }
 
   button {
     display: none;
   }
 
+  .letter-list li:nth-child(n+2) {
```

Comment:
It grabs all the letters apart from the first one so that we don't display them on the webview (as opposed to the printview).